swatch-core/src/main/java/org/candlepin/subscriptions/db/model/HardwareMeasurement.java

```diff
@@ -31,9 +32,13 @@
 @Getter
 @Setter
 public class HardwareMeasurement implements Serializable {
+
+  @NotNull
```

Comment:
`@NotNull` on `int` a bit redundant, since Java will default to `0` but I don't mind.